MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,51 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace magneticFieldParabolicPortable {
+
+    struct Parameters {
+      // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
+      // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+      static constexpr float c1 = 3.8114;
+      static constexpr float b0 = -3.94991e-06;
+      static constexpr float b1 = 7.53701e-06;
+      static constexpr float a = 2.43878e-11;
+      static constexpr float max_radius2 = 13225.f;  // tracker radius
+      static constexpr float max_z = 280.f;          // tracker z
+    };
+
+    template <typename Vec3>
+    constexpr float Kr(Vec3 const& vec) {
+      return Parameters::a * (vec(0) * vec(0) + vec(1) * vec(1)) + 1.f;
+    }
+
+    template <typename Vec3>
+    constexpr float B0Z(Vec3 const& vec) {
+      return Parameters::b0 * vec(2) * vec(2) + Parameters::b1 * vec(2) + Parameters::c1;
+    }
+
+    template <typename Vec3>
+    constexpr bool isDefined(Vec3 const& vec) {
+      return ((vec(0) * vec(0) + vec(1) * vec(1)) < Parameters::max_radius2 && fabs(vec(2) < Parameters::max_z));
```

Comment:
I think this should be
```suggestion
      return ((vec(0) * vec(0) + vec(1) * vec(1)) < Parameters::max_radius2 && fabs(vec(2)) < Parameters::max_z);
```
?

Reply:
By the way, technically, `fabs(...)` will break if we add the SYCL backend.

Reply:
Fixed: https://github.com/cms-sw/cmssw/pull/44360/commits/8d93f61cc051bd4b2a39d6559170dd00f6a2bcef